test/test_command_line.py

```diff
@@ -0,0 +1,32 @@
+from datascroller import command_line
```

Comment:
simple tests to make sure we can parse valid arguments. We should probably add negative tests here as well.